test/notebooks/test_notebooks.py

```diff
@@ -139,4 +139,12 @@ def test_mitigating_disparities_in_ranking_from_binary_data():
 def test_binary_classification_with_the_uci_credit_card_default_dataset():
     nb_name = "Binary Classification with the UCI Credit-card Default Dataset"
     test_values = {}
+    test_values["GridSearch_17"] = ScrapSpec(
```

Comment:
You could directly initialize the dict with these values... not a big deal I guess